scripts/amalgamation.py

```diff
@@ -201,7 +208,8 @@ def generate_duckdb_hpp(header_file):
 
         hfile.write("#pragma once\n")
         hfile.write("#define DUCKDB_AMALGAMATION 1\n")
-        hfile.write("#define DUCKDB_SOURCE_ID \"%s\"\n" % git_commit_hash().decode('utf8'))
+        hfile.write("#define DUCKDB_SOURCE_ID \"%s\"\n" % git_commit_hash())
+        hfile.write("#define DUCKDB_DEV_VERSION \"%s\"\n" % git_dev_version())
```

Comment:
Maybe it should not only be suitable for designating the `DEV` version, but the DuckDB version in general.

```suggestion
        hfile.write("#define DUCKDB_VERSION \"%s\"\n" % git_dev_version())
```

Reply:
Fair enough.